_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation-filter-popin/smart-shopping-campaign-creation-popin-recap.vue

```diff
@@ -139,6 +139,14 @@ export default {
       // TODO also check with PM how it is supposed to be presented
       return this.totalProducts;
     },
+    endDate() {
+      const endDate = new Date(this.newCampaign.endDate).getFullYear();
+      const todayYear = new Date().getFullYear();
+      if (endDate - 10 > todayYear) {
```

Comment:
Logic duplicated from another file. Time to refacto this as a common method.

@amaury-hanser also suggested to add a comment explaining why we have this check